frontend/components/icon/IconSignal.vue

```diff
@@ -0,0 +1,10 @@
+
+<template name="iconsignal">
```

Comment:
We actually don't need this component, @Mister-Tofu-tech :) My suggestion would be to use the built in `<Icon />` component via Nuxt Icon. You can pass in an icon that you find on [Icones](https://icones.js.org/) in the following way, with `simple-icons:signal` being one I'd suggest:

```
<Icon name="simple-icons:signal" size="1.5em" />
```

This can be directly dropped into `ModalSharePage`. We do not need a separate component for it :)

Reply:
Hey @Mister-Tofu-tech 👋 Given the changes recently in #868, the code for this would now be:

```ts
<Icon :name="IconMap.SIGNAL" size="1.5em" />
```